GCC 11 pedantry.

```
read_config.c: In function ‘read_config’:
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:116:17: note: in expansion of macro ‘getval_yn’
  116 |                 getval_yn("8bit", o->format, XMP_FORMAT_8BIT);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:116:17: note: in expansion of macro ‘getval_yn’
  116 |                 getval_yn("8bit", o->format, XMP_FORMAT_8BIT);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:117:17: note: in expansion of macro ‘getval_yn’
  117 |                 getval_yn("mono", o->format, XMP_FORMAT_MONO);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:117:17: note: in expansion of macro ‘getval_yn’
  117 |                 getval_yn("mono", o->format, XMP_FORMAT_MONO);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:118:17: note: in expansion of macro ‘getval_yn’
  118 |                 getval_yn("filter", o->dsp, XMP_DSP_LOWPASS);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:118:17: note: in expansion of macro ‘getval_yn’
  118 |                 getval_yn("filter", o->dsp, XMP_DSP_LOWPASS);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:119:17: note: in expansion of macro ‘getval_yn’
  119 |                 getval_yn("loop", o->loop, 1);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:119:17: note: in expansion of macro ‘getval_yn’
  119 |                 getval_yn("loop", o->loop, 1);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:120:17: note: in expansion of macro ‘getval_yn’
  120 |                 getval_yn("reverse", o->reverse, 1);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:120:17: note: in expansion of macro ‘getval_yn’
  120 |                 getval_yn("reverse", o->reverse, 1);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:130:17: note: in expansion of macro ‘getval_yn’
  130 |                 getval_yn("amiga", o->amiga_mixer, 1);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:130:17: note: in expansion of macro ‘getval_yn’
  130 |                 getval_yn("amiga", o->amiga_mixer, 1);
      |                 ^~~~~~~~~
read_config.c: In function ‘parse_modconf’:
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:238:17: note: in expansion of macro ‘getval_yn’
  238 |                 getval_yn("filter", o->dsp, XMP_DSP_LOWPASS);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:238:17: note: in expansion of macro ‘getval_yn’
  238 |                 getval_yn("filter", o->dsp, XMP_DSP_LOWPASS);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:239:17: note: in expansion of macro ‘getval_yn’
  239 |                 getval_yn("loop", o->loop, 1);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:239:17: note: in expansion of macro ‘getval_yn’
  239 |                 getval_yn("loop", o->loop, 1);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:240:17: note: in expansion of macro ‘getval_yn’
  240 |                 getval_yn("mono", mono, 1);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:240:17: note: in expansion of macro ‘getval_yn’
  240 |                 getval_yn("mono", mono, 1);
      |                 ^~~~~~~~~
read_config.c:107:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  107 |             else w &= ~(y); continue; } }
      |             ^~~~
read_config.c:241:17: note: in expansion of macro ‘getval_yn’
  241 |                 getval_yn("reverse", o->reverse, 1);
      |                 ^~~~~~~~~
read_config.c:107:29: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  107 |             else w &= ~(y); continue; } }
      |                             ^~~~~~~~
read_config.c:241:17: note: in expansion of macro ‘getval_yn’
  241 |                 getval_yn("reverse", o->reverse, 1);
      |                 ^~~~~~~~~
read_config.c:111:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  111 |             else w = -1; continue; } }
      |             ^~~~
read_config.c:246:17: note: in expansion of macro ‘getval_tristate’
  246 |                 getval_tristate("fixloop", o->fixloop);
      |                 ^~~~~~~~~~~~~~~
read_config.c:111:26: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  111 |             else w = -1; continue; } }
      |                          ^~~~~~~~
read_config.c:246:17: note: in expansion of macro ‘getval_tristate’
  246 |                 getval_tristate("fixloop", o->fixloop);
      |                 ^~~~~~~~~~~~~~~
read_config.c:111:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  111 |             else w = -1; continue; } }
      |             ^~~~
read_config.c:247:17: note: in expansion of macro ‘getval_tristate’
  247 |                 getval_tristate("fx9bug", o->fx9bug);
      |                 ^~~~~~~~~~~~~~~
read_config.c:111:26: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  111 |             else w = -1; continue; } }
      |                          ^~~~~~~~
read_config.c:247:17: note: in expansion of macro ‘getval_tristate’
  247 |                 getval_tristate("fx9bug", o->fx9bug);
      |                 ^~~~~~~~~~~~~~~
read_config.c:111:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  111 |             else w = -1; continue; } }
      |             ^~~~
read_config.c:248:17: note: in expansion of macro ‘getval_tristate’
  248 |                 getval_tristate("vblank", o->vblank);
      |                 ^~~~~~~~~~~~~~~
read_config.c:111:26: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  111 |             else w = -1; continue; } }
      |                          ^~~~~~~~
read_config.c:248:17: note: in expansion of macro ‘getval_tristate’
  248 |                 getval_tristate("vblank", o->vblank);
      |                 ^~~~~~~~~~~~~~~
read_config.c:111:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  111 |             else w = -1; continue; } }
      |             ^~~~
read_config.c:249:17: note: in expansion of macro ‘getval_tristate’
  249 |                 getval_tristate("amiga", o->amiga_mixer);
      |                 ^~~~~~~~~~~~~~~
read_config.c:111:26: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  111 |             else w = -1; continue; } }
      |                          ^~~~~~~~
read_config.c:249:17: note: in expansion of macro ‘getval_tristate’
  249 |                 getval_tristate("amiga", o->amiga_mixer);
      |                 ^~~~~~~~~~~~~~~
```

--- a/src/read_config.c
+++ b/src/read_config.c
@@ -104,11 +104,13 @@ int read_config(struct options *o)
 
 #define getval_yn(x,w,y) { \
 	if (!strcmp(var,x)) { if (get_yesno (val)) w |= (y); \
-	    else w &= ~(y); continue; } }
+	    else w &= ~(y); \
+	    continue; } }
 
 #define getval_tristate(x,w) { \
 	if (!strcmp(var,x)) { if (get_yesno (val)) w = 1; \
-	    else w = -1; continue; } }
+	    else w = -1; \
+	    continue; } }
 
 #define getval_no(x,y) { \
 	if (!strcmp(var,x)) { y = atoi (val); continue; } }

